src/bitcoin-chainstate.cpp

```diff
@@ -42,6 +43,8 @@
 
 int main(int argc, char* argv[])
 {
+    LogInstance().DisableLogging();
```

Comment:
It's mostly added so that `DisableLogging()` isn't dead code; all it really achieves is a minor performance improvement. Added some comments that maybe clarifies things.